.changeset/six-crabs-remember.md

```diff
@@ -0,0 +1,5 @@
+---
+'@openfn/language-rapidpro': minor
```

Comment:
This is actually a major change, because we're removing the `axios` export (you used to be able to use axios in job code directly).

We need to mention this in the release notes.